bookwyrm/models/bookwyrm_export_job.py

```diff
@@ -49,21 +51,22 @@ def start_export_task(**kwargs):
     job.save(update_fields=["export_data"])
 
 
-def tar_export(json_data: str, user, f):
-    f.open("wb")
-    with BookwyrmTarFile.open(mode="w:gz", fileobj=f) as tar:
+def tar_export(json_data: str, user, file):
+    """wrap the export information in a tar file"""
+    file.open("wb")
+    with BookwyrmTarFile.open(mode="w:gz", fileobj=file) as tar:
         tar.write_bytes(json_data.encode("utf-8"))
 
         # Add avatar image if present
         if getattr(user, "avatar", False):
             tar.add_image(user.avatar, filename="avatar")
 
-        editions, books = get_books_for_user(user)
+        editions = get_books_for_user(user)
```

Comment:
Isn't this going to return a tuple stored in `editions`, rather than the original behavior that stores just the first value of the tuple?

Reply:
hmm, yes. Looks like I was a little hasty "fixing" pylint complaints.